engine/graphics/src/null/graphics_null.cpp

```diff
@@ -811,8 +811,12 @@ namespace dmGraphics
 
     static ShaderDesc::Language NullGetShaderProgramLanguage(HContext context)
     {
-#if defined(DM_GRAPHICS_NULL_SHADER_LANGUAGE)
-        return ShaderDesc:: DM_GRAPHICS_NULL_SHADER_LANGUAGE ;
+#if defined(DM_PLATFORM_VENDOR)
+        #if defined(DM_GRAPHICS_NULL_SHADER_LANGUAGE)
+            return ShaderDesc:: DM_GRAPHICS_NULL_SHADER_LANGUAGE ;
+        #else
+            #error "You must define the platform default shader language using DM_GRAPHICS_NULL_SHADER_LANGUAGE"
```

Comment:
Add a compile error if the language isn't set (to avoid debugging until I figure it out again)